README.md

```diff
@@ -135,17 +135,15 @@ To install a specific version, visit [releases](https://github.com/PowerShell/Po
 
 For more information on how and why we built this dashboard, check out this [blog post](https://devblogs.microsoft.com/powershell/powershell-open-source-community-dashboard/).
 
-## Chat Room
+## Chat
 
 Want to chat with other members of the PowerShell community?
 
 We have a Gitter Room which you can join below.
 
 [![Join the chat](https://img.shields.io/static/v1.svg?label=chat&message=on%20gitter&color=informational&logo=gitter)](https://gitter.im/PowerShell/PowerShell?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
 
-There is also the community driven PowerShell Slack Team which you can sign up for at [Slack].
-
-[Slack]: http://slack.poshcode.org
+There is also the community driven PowerShell Virtual User Group, which you can join by following the instructions at <https://poshcode.org/>.
```

Comment:
Shouldn't we be listing the aka.ms links here as well?

https://aka.ms/psslack

https://aka.ms/psdiscord

Reply:
https://aka.ms/psslack had expired but is now updated thanks to @TylerLeonhardt